Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -17,7 +17,7 @@ extension MongoCollection {
      */
     @discardableResult
     public func bulkWrite(_ requests: [WriteModel], options: BulkWriteOptions? = nil) throws -> BulkWriteResult? {
-        if requests.isEmpty {
+        guard !requests.isEmpty else {
```

Comment:
`guard !...` stood out in contrast to most of the other changes in this PR -- but I think it's still preferable to something like `guard requests.count > 0`.

Side observation while I was looking into this: Both [`Array.isEmpty`]((https://developer.apple.com/documentation/swift/array/1688398-isempty)) and `.count` are constant-time. [Collection.count](https://developer.apple.com/documentation/swift/array/2949997-count) is either O(1) or O(n) depending on whether the implementation supports random access, which Array does.

Anywho, this is fine as-is. Just wanted to record my thought process :)

Reply:
there's been [some discussion](https://forums.swift.org/search?q=isnotempty) of adding `.isNotEmpty` in the past... but verdict seems to be it's not gonna happen

Reply:
It is always odd to me to see things like negations of library functions like `isEmpty -> isNonEmpty` being rejected. The usual arguments of trying to only add functions when absolutely necessary just doesn't convince me. There _is_ a cognitive burden to having to do `!isEmpty()` because you have to catch the `!`, which I'm sure we have all at some point missed. `isNonEmpty()` just reads so much better because its just straight up natural language.

In either case, @jmikola, feel free to resolve this since I think we want to follow the insight Kaitlin got from Jeff. :).